app/controllers/admin/participants/change_cohort_controller.rb

```diff
@@ -4,14 +4,14 @@ module Admin::Participants
   class ChangeCohortController < Admin::BaseController
     def edit
       @participant_profile = retrieve_participant_profile
-      @latest_induction_record = @participant_profile.latest_induction_record
+      @relevant_induction_record = Induction::FindBy.new(participant_profile: @participant_profile).call
```

Comment:
Fyi these were designed to be invoked without `.new` like `Induction::FindBy.call(participant_profile: @participant_profile)` but it will work either way

Reply:
Ah yes, fixed and force pushed. Thanks 👍🏽